src/gleam_community/ansi.gleam

```diff
@@ -93,6 +97,7 @@ import gleam/bitwise
 import gleam/int
 import gleam/list
 import gleam/string
+import gleam_community/colour.{Colour} as gc_colour
```

Comment:
I don't know how much you want my personal opinions to factor into review, but personally I don't alias imports _ever_ unless there's a clash (eg the `gleam_community/maths` stuff clashing with stdlib).

Reply:
I generally agree, for this the alias comes from the `colour` function also having a parameter named `colour` which then causes the clash. I figured better to alias the import than come up with a different parameter that may be more confusing.